doc/application/index.rst

```diff
@@ -580,7 +580,7 @@ targeting this board by specifying the location of your custom board
 information with the ``-DBOARD_ROOT`` parameter to the CMake
 build system::
 
-   cmake -DBOARD=<board name> -DBOARD_ROOT=<path to boards> ..
+   cmake -DBOARD=<board name> -DBOARD_ROOT=<path the directory containig a "boards" directory> ..
```

Comment:
Language nit: "path to directory containing a boards/ directory"

Reply:
... also fixes the misspelling of "containing"  :)